### PR TITLE
Release 2.5

### PR DESCRIPTION
**Pull request by ariya**

### CODE REVIEW

**Comment by ariya** [comment]
For detailed performance numbers, see #1246.

**Comment by ariya** [comment]
Any objection to finally release it this evening? @mikesherov @ikarienator

**Comment by mikesherov** [comment]
:+1:

File: ChangeLog [hunk 1]
```diff
@@ -1,3 +1,18 @@
+2015-07-31: Version 2.5.0
+
+      * Run the test suite in a browser environment (issue 1004)
+      * Ensure a comma between imported default binding and named imports (issue 1046)
+      * Distinguish `yield` as a keyword vs an identifier (issue 1186)
+      * Support ES5 meta property `new.target` (issue 1203)
```
Comment (azu) [inline ChangeLog]: ES5 -> ES6?

Comment (ariya) [inline ChangeLog]: > ES5 -> ES6?

Good catch, thanks! I'll fix it shortly.

**Pull request closed without merging**